src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  background: #fff;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon--big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input--big {
+  text-indent: 62px;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, sans-serif;

```

Comment:
```suggestion
  font-family: inherit;
```
You can inherit it from HTML everywhere

Reply:
Not fixed. Doesn't it work the same with inherit?
![image](https://user-images.githubusercontent.com/96409563/230161726-f4ee2e54-d828-45aa-959a-3c07729ea987.png)

Reply:
yes, tried but it did not work